### ISSUE TITLE
[RFC] `hit new` without arguments should choose the only assigned issue and create a branch for it

### ISSUE THREAD

**Issue by chshersh** [kowainik/hit-on#116]
Often enough, I have only one issue assigned to me in a project. In that case, my workflow looks like this:

1. `hit issue` to see the issue number
2. `hit new 1234` to create a branch.

I propose to patch `hit new` command, so it automatically chooses the issue if you have only one issue assigned to you. With the following handling of exceptional cases:

* If you have multiple issues assigned to you, start an interactive process to ask which one you want by displaying all issues assigned to you
* If you don't have any issues assigned to you, ask to create a new issue (after #65 is implemented)

Inside this interactive process, it should be possible to write a raw text of the branch name if you don't want to attach the branch to an issue.

**Comment by vrom911** [comment]
@chshersh Do you think that it makes sense to have this behaviour for the `new` command only when `--me` option is specified? Or it is okay to do so without this option?

**Comment by chshersh** [comment]
@vrom911 I think it's a good idea to be more explicit 👍 I agree with the `--me` option
